lib/tasks/data.rake

```diff
@@ -44,6 +44,32 @@ namespace :db do # rubocop:disable Metrics/BlockLength
   task add_friendlyid_organisation_slugs: :environment do
     SetOrganisationSlugsJob.perform_now
   end
+
+  desc "Set other_start_date_details for vacancies with starts_asap"
+  task set_other_start_date_details: :environment do
```

Comment:
Can these 2 tasks be just one? they take time when run in production, so better if they are done in one go